sympy/solvers/solveset.py

```diff
@@ -234,6 +231,69 @@ def _invert_complex(f, g_ys, symbol):
     return (f, g_ys)
 
 
+def _invert_abs(f, g_ys, symbol):
+    """Helper function for inverting absolute value functions"""
```

Comment:
In the docstring you should briefly explain how this sub routine works, this is not required for other `_invert_*` because they are largely trivial or the code is self explanatory.